about.html

```diff
@@ -5,10 +5,120 @@
+                <div class="about__contact flex column">
+                    <p class="about--peragraph">
+                        Please contact us per Email for any further questions about CC Global Summit 2015!
+                    </p>
+                    <a href="#">info@itlive.iq</a>
+                </div>
+            </div>
+            <div class="about__logo flex column">
+                <h2 class="section--title">IT.LIVE IRAQ TECH SUMMIT 2019 Logo</h2>
+                <p class="about--peragraph">The logo of IT.LIVE IRAQ TECH SUMMIT 2019 was decided through the logo
+                    competition
+                    from 8. June to 7. July.</p>
+                <img src="images/brands/it_live.png" alt="" class="globale__summit__logo">
+            </div>
+            <div class="past__summit flex column">
+                <h2 class="section--title">See the past IT.LIVE Summits</h2>
+                <p class="about--peragraph">Take a look at the last two IT.LIVE Summits which took place in Baghdad.</p>
+                <div class="summit__group flex column">
+                    <div class="summit">
+                        <img src="images/it_live_2018.jpeg" alt="it live 2018 Iraq, Baghdad">
+                        <span class="image__overlay"></span>
+                        <h3 class="summit__date">2018</h3>
+                        <p class="summit__location">Iraq, Baghdad</p>
+                    </div>
+                    <div class="summit">
+                        <img src="images/it_live_2017.jpeg" alt="it live 2017 Iraq, Baghdad">
+                        <span class="image__overlay"></span>
+                        <h3 class="summit__date">2017</h3>
+                        <p class="summit__location">Iraq, Baghdad</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="partners flex column bg-dark" id="partners">
+            <h2 class="section--title">Partners</h2>
+            <ul class="nav--brands">
+                <li><img src="images/brands/careem.svg" alt="mozilla co. brand logo"></li>
+                <li><img src="images/brands/google.svg" alt="google co. brand logo"></li>
+                <li><img src="images/brands/naver.svg" alt="naver co. brand logo"></li>
+                <li><img src="images/brands/daumkakao.svg" alt="daumkakao co. brand logo"></li>
+                <li><img src="images/brands/airbnb.svg" alt="airbnb co. brand logo"></li>
+            </ul>
+        </section>
+    </main>
+
+    <footer class="footer flex">
+        <div class="footer__group">
+            <img src="images/brands/it_live.png" alt="creative commones korea logo">
+        </div>
+        <div class="footer__group span-2">
+            <p>"Like Never Before"</p>
+            <p>2019 The largest Iraqi forum for information technology in all its disciplines.</p>
+        </div>
+    </footer>
+
+    <script src="main.js"></script>
```

Comment:
- [ ] Please, consider removing the partner's section here, as it is in the original design.
- [ ] Please, consider that on the about page the footer should be dark, as it is required.

> Your version:
<img width="1409" alt="Screen Shot 2022-03-13 at 00 07 24" src="https://user-images.githubusercontent.com/49064106/158033414-77a85b1f-92da-415a-8430-9d584b729684.png">

> Original version:
<img width="863" alt="Screen Shot 2022-03-13 at 00 11 57" src="https://user-images.githubusercontent.com/49064106/158033431-ab000b80-d1ab-41f1-8ecd-09ba797ca097.png">